test/Test/Database/LSMTree/Internal/Lookup.hs

```diff
@@ -150,8 +150,8 @@ prop_bloomQueriesModel dats =
 bloomQueriesModel :: [Set SerialisedKey] -> [SerialisedKey] -> [(RunIx, KeyIx)]
 bloomQueriesModel blooms ks = [
       (rix, kix)
-    | (rix, b) <- rs'
-    , (kix, k) <- ks'
+    | (kix, k) <- ks'
+    , (rix, b) <- rs'
     , Set.member k b
     ]
```

Comment:
The order has to be changed in `prepLookupsModel` as well

Reply:
Done.